app/elements/candidates/invite-candidates.html

```diff
@@ -78,14 +74,13 @@
 				}
 			},
 			_addCandidate: function(){
-				if (this.challenge){
-					this.push('candidates', {'Challenge': this.challenge.Key});
+				if (this.challenges.length == 1){
+					this.push('candidates', {'Challenge': this.challenges[0].Key});
```

Comment:
No, it doesn't.
![screenshot from 2015-09-08 16-02-01](https://cloud.githubusercontent.com/assets/9076897/9735367/f7364774-5642-11e5-981e-5a66dbaba72e.png)